docs/tutorials/TUTORIAL_03_TITLES_AND_COLORS.md

```
@@ -0,0 +1,110 @@
+# <img src="../assets/logo.png"> Tutorial 3 – Titles and Colors
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)